### DISCUSSION TITLE
I have a personal copilot.My company will also give me a copilot license.What's going to happen here?

### DISCUSSION THREAD

**Discussion by SilenceZen** [community/community#55909] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I wanna some  explanation of what might happen

**Comment by Akash1134** [comment, marked as answer]
Hi @SilenceZen , 

Your personal Copilot license is subjected to be used for personal projects and not in a business environment. So, there should not be any problem with you getting a business license seat unless you are not using your personal license in an Org setup.  

Your Organization owner will have policies setup and if they want to map your current license in usage for the business, they can update the same. For more details, you can check out this [documentation](https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-copilot-in-your-enterprise#enforcing-a-policy-to-manage-the-use-of-github-copilot-for-business-in-your-enterprise). 

Hope that helps!

**Reply by SilenceZen** [reply]
Yes.  I mean I have myself copilot.  but my company will give me a business license. How to deal with my copilot

**Reply by SilenceZen** [reply]
Thinks. I cancel my copilot and I get refund
